torch/utils/data/_utils/collate.py

```diff
@@ -61,7 +61,7 @@ def default_collate(batch):
         return batch
     elif isinstance(batch[0], container_abcs.Mapping):
         return {key: default_collate([d[key] for d in batch]) for key in batch[0]}
-    elif isinstance(batch[0], tuple) and hasattr(batch[0], '_fields'):  # namedtuple
+    elif isinstance(batch[0], tuple):
         return type(batch[0])(*(default_collate(samples) for samples in zip(*batch)))
```

Comment:
oh and the tuple ctor can not be called this way, so this should be another elif.